prow/config.yaml

```diff
@@ -685,7 +685,7 @@ presubmits:
     trigger: "(?m)^/test( all| pull-cluster-registry-verify-gensrc),?(\\s+|$)"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:v20171214-ae38d82af-0.8.1
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180123-1260ba9fc-1.8
```

Comment:
you might want to use `-master` version here to keep updated with latest go version we bumped

Reply:
Thanks, that seems reasonable. Done.